src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Attributes/JsonRequiredAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies the property is required when deserializing.
```

Comment:
It might make sense to add a remark clarifying that an explicitly specified `null` token in JSON will not trigger a validation error.